modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
+examplecluster-compute-0          Running                          165m    openshift-compute-0         baremetalhost:///openshift-machine-api/openshift-compute-0/3d685b81-7410-4bb3-80ec-13a31858241f         provisioned
+examplecluster-compute-1          Running                          165m    openshift-compute-1         baremetalhost:///openshift-machine-api/openshift-compute-1/0fdae6eb-2066-4241-91dc-e7ea72ab13b9         provisioned
+----
+<1> This is the control plane machine for the unhealthy node, `examplecluster-control-plane-2`.
+
+.. Save the machine configuration to a file on your file system:
++
+[source,terminal]
+----
+$ oc get machine examplecluster-control-plane-2 \ <1>
+    -n openshift-machine-api \
+    -o yaml \
+    > new-master-machine.yaml
+----
+Specify the name of the control plane machine for the unhealthy node.
+
+.. Edit the `new-master-machine.yaml` file that was created in the previous step to assign a new name and remove unnecessary fields.
+
+... Remove the entire `status` section:
++
+[source,yaml]
+----
+status:
+  addresses:
+  - address: ""
+    type: InternalIP
+  - address: fe80::4adf:37ff:feb0:8aa1%ens1f1.373
+    type: InternalDNS
+  - address: fe80::4adf:37ff:feb0:8aa1%ens1f1.371
+    type: Hostname
+  lastUpdated: "2020-04-20T17:44:29Z"
+  nodeRef:
+    kind: Machine
+    name: fe80::4adf:37ff:feb0:8aa1%ens1f1.372
+    uid: acca4411-af0d-4387-b73e-52b2484295ad
+  phase: Running
+  providerStatus:
+    apiVersion: machine.openshift.io/v1beta1
+    conditions:
+    - lastProbeTime: "2020-04-20T16:53:50Z"
+      lastTransitionTime: "2020-04-20T16:53:50Z"
+      message: machine successfully created
+      reason: MachineCreationSucceeded
+      status: "True"
+      type: MachineCreation
+    instanceId: i-0fdb85790d76d0c3f
+    instanceState: stopped
+    kind: Machine
+----
+
+. Change the `metadata.name` field to a new name.
++
+It is recommended to keep the same base name as the old machine and change the ending number to the next available number. In this example, `examplecluster-control-plane-2` is changed to `examplecluster-control-plane-3`.
++
+For example:
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  ...
+  name: examplecluster-control-plane-3
+  ...
+----
+
+.. Remove the `spec.providerID` field:
++
+[source,yaml]
+----
+  providerID: baremetalhost:///openshift-machine-api/openshift-control-plane-2/3354bdac-61d8-410f-be5b-6a395b056135
+----
+
+.. Remove the `metadata.annotations` and `metadata.generation` fields:
++
+[source,yaml]
+----
+  annotations:
+    machine.openshift.io/instance-state: externally provisioned
+  ...
+  generation: 2
+----
+
+.. Remove the `spec.conditions`, `spec.lastUpdated`, `spec.nodeRef` and `spec.phase' fields:
```

Comment:
```suggestion
.. Remove the `spec.conditions`, `spec.lastUpdated`, `spec.nodeRef` and `spec.phase` fields:
```

Reply:
This is where I found a "hidden backtick". I THINK it's a vscode/codium thing (trying to "close" the backticks) but I'm not sure at this point. Will take a look at the file with vim to double-check.